src/components/ImageGeneration/FieldSkeleton.tsx

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable jsx-a11y/alt-text */
+/* eslint-disable @next/next/no-img-element */
+import type { CSSProperties } from 'react';
+
+export const FieldSkeleton = ({
+  width,
+  height,
+  sx,
+}: {
+  width: number;
+  height: number;
+  sx?: CSSProperties;
+}) => {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        width,
+        height,
+        borderRadius: height,
+        backgroundColor: '#74747454',
+        ...(sx && sx),
```

Comment:
this can crash the app

Reply:
@oktapodia  Is this a proper workaround?
![image](https://github.com/user-attachments/assets/4d595659-00e1-4882-ac2c-6e0cf722c64b)